README.md

```diff
@@ -1,21 +1,75 @@
+
+
+[sales@syncfusion.com](mailto:sales@syncfusion.com?Subject=Syncfusion Blazor Accordion - GiHub) | [www.syncfusion.com](https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples) | Toll Free: 1-888-9 DOTNET
```

Comment:
This won't work
![image](https://user-images.githubusercontent.com/61148953/161201384-92fabb2e-f6f1-4ed2-869f-bd7892b6325e.png)

use like this 

<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />

<p align="center">
  <a href="mailto:sales@syncfusion.com?Subject=Subject=Syncfusion Blazor Accordion - GiHub" target="_top">sales@syncfusion.com</a>   | <a href="https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=blazor-accordion-github-samples">www.syncfusion.com</a> | Toll Free: 1-888-9 DOTNET <br>
</p>